src/plugins/saved_objects_management/public/index.ts

```diff
@@ -23,7 +23,13 @@ export type {
 } from './services';
 export { SavedObjectsManagementAction } from './services';
 export type { ProcessedImportResponse, FailedImport } from './lib';
-export { processImportResponse } from './lib';
+export {
+  processImportResponse,
+  getSavedObjectLabel,
+  getAllowedTypes,
+  parseQuery,
+  getTagFindReferences,
+} from './lib';
```

Comment:
We want to avoid utility exports. Consumers should use to SOM contract instead. Most of the functions you exported are already available here:

https://github.com/elastic/kibana/blob/606d9c2649463db2501ba5cf7af9af6b67855ff7/src/plugins/saved_objects_management/public/plugin.ts#L123-L127